### ISSUE TITLE
Minor issue: Column header not announced for "Step #" column in Word export

### ISSUE THREAD

**Issue by sjoehl** [maureendmccleary/functional-test-tool#53]
When navigating by column in the body of the results table, the column header of "Step number" is not announced by JAWS. NVDA does not announce any column headers.
